content/en/getting_started/dashboards/_index.md

```diff
@@ -0,0 +1,134 @@
+
+Dashboards can also help you track down persistent problems and fix them. _Troubleshooting dashboards_ often start as a scratch pad of things you know, and gradually build as you discover more. For example, start with a graph or widget from another dashboard or view that shows a problem-you'll be drilling down from there to find your solution.
+
+## Explore out-of-the-box dashboards
+
+Datadog provides many out-of-the-box dashboards for features and integrations. For the infrastructure you monitor, check out the out-of-the-box dashboards that are provided with Datadog:
+
+1. Go to **Dashboards > Dashboards list** and search for the name of an integration you have added, for example `Redis`, or a feature you use, such as `RUM`. 
+2. Browse the search results for dashboards marked *Preset* and see if at least some of the graphs show the answers you're looking for.
+3. Explore the links in the out-of-the-box dashboard's title drop-down to find more information about how people are using them.
+
+## Start by reusing good things
+
+A common way to start a dashboard is by encountering a similar dashboard already in use, and adjusting it to suit your needs. If you find a dashboard that answers many of the questions you want your dashboard to answer: 
+
+1. Clone it by opening the dashboard and selecting **Clone dashboard** from the Settings menu (the gear icon on the right-hand side). This creates an unlinked copy of the dashboard; changes you make in the new copy don't affect the source widget.
+2. Edit the clone by opening it and clicking **Edit widgets**. 
+3. Delete widgets you don't use by selecting **Delete** from the widget's Settings menu.
+4. Move things around to suit your needs. Groups and individual widgets can be dragged and dropped into new locations in the dashboard.
+5. Copy in widgets you like from other dashboards by hovering over the widget and typing Command + C (Ctrl + C on Windows). Paste it into your dashboard by opening the dashboard and typing Command + V (Ctrl + V on Windows).
+5. Use the **Export to Dashboard** option provided by many Datadog views for data they show. For example, the Logs Explorer and Log Analytics views have share options to export logs lists and metrics to dashboards.
+
+## Figuring out what a metric means
+
+Through integrations, Datadog collects [metrics][3] about your infrastructure and applications. The collected metrics are documented in the integration's Readme files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric means, look it up in the Integrations docs. 
```

Comment:
```suggestion
Through integrations, Datadog collects [metrics][3] from your infrastructure and applications. The collected metrics are documented in the integration's README files. If you encounter a metric in the [Metrics Explorer][4] or as you're creating a dashboard, and you want to know what the metric is, look it up in the Integrations docs. 
```

Reply:
Probably helpful to drop a link to an existing integrations doc with metrics for context